scipy/optimize/_minimize.py

```diff
@@ -38,6 +38,17 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
              callback=None, options=None):
     """
     Minimization of scalar function of one or more variables.
+     
+    In general, the optimization problems are of the form:
+    
+    minimize f(x)
+    subect to: 
+        g_i(x) >= 0, i = 1,...,m
+        h_i(x)  = 0, i = 1,...,p
+        
+    Where x is a vector of one or more variables.
+    g_i(x) are the inequality constraints.
+    h_i(x) are the equality constrains.
```

Comment:
``x, ``g_i(x)`` and ``h_i(x)`` need to be surrounded by double backticks to get html formatting right. Like:

    ``g_i(x)``